### PR TITLE
Required modification to support bal health CLI

### PR DESCRIPTION
**Pull request by isuruh15**

### CODE REVIEW

File: components/commons/src/main/java/org/wso2/healthcare/codegen/tool/framework/commons/config/ToolConfig.java [hunk 1]
```diff
@@ -61,4 +62,10 @@ public interface ToolConfig {
      * @param targetDir path to the target directory
      */
     void setTargetDir(String targetDir);
+    
+    public String getSpecBasePath();
```
Comment (sameeragunarathne) [inline components/commons/src/main/java/org/wso2/healthcare/codegen/tool/framework/commons/config/ToolConfig.java:66]:
```suggestion
   String getSpecBasePath();
```

Comment (sameeragunarathne) [reply components/commons/src/main/java/org/wso2/healthcare/codegen/tool/framework/commons/config/ToolConfig.java:66]: No need to have the public access modifier in the interface methods

Comment (sameeragunarathne) [reply components/commons/src/main/java/org/wso2/healthcare/codegen/tool/framework/commons/config/ToolConfig.java:66]: Add javadocs for the newly added interface methods

Comment (isuruh15) [reply components/commons/src/main/java/org/wso2/healthcare/codegen/tool/framework/commons/config/ToolConfig.java]: fixed in 
ab4762c

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**